### ISSUE TITLE
NOALBS and Kiloview p2 with Nginx doesn't change the scene when disconnecting

### ISSUE THREAD

**Issue by FkManu** [715209/nginx-obs-automatic-low-bitrate-switching#120]
Hi, I have a little problem with the switcher:
I connected a kiloview p2 (in bonding) to an Nginx server on a vps, it creates an always active channel with the Nginx server that constantly marks 0 bit/s. The switcher works well on drops, but when the encoder is turned off the scene change is blocked on the "live'' scene, without more switching to the "low'' scene, probably because the signal is always constantly active at 0 bit/s, I tried to raise the trigger threshold but nothing changes anyway, is it possible that it is a bug? 
Usually in my classic RTMP-Nginx system when the signal is off the bot replies with '' No connection :( '' usually in my classic RTMP-Nginx system when the signal is off the bot replies with `` No connection :( '' instead with the encoder signal off it replies ''nginx: 0''

![chatterino_cXdSl9Zzgp](https://user-images.githubusercontent.com/79100552/190253559-2796b7f5-ee39-4722-8ce5-0a03094dccf8.png)
![image_2022-09-14_22-16-46](https://user-images.githubusercontent.com/79100552/190253865-718cc0e5-fe10-4c39-8df3-5ca44114aa26.png)

**Comment by DigibitDK** [comment]
What Noalbs-version are you using?
As far as I can tell, your have 'idle_streams on;' in NGINX right?

**Comment by FkManu** [comment]
> What Noalbs-version are you using? As far as I can tell, your have 'idle_streams on;' in NGINX right?

I am currently using version 2.7.2 (the latest) and on Nginx I have 'idle_streams on;' because otherwise it gives me problems with the kilolink server to which I connected Nginx

**Comment by DigibitDK** [comment]
Just to be clear. The issue that you experience is that it stays on your live-scene, when you disconnect your kiloview right?
But otherwise scene-switching is working fine. It will switch from live to low-bitrate on low-bitrate?

**Comment by FkManu** [comment]
> Just to be clear. The issue that you experience is that it stays on your live-scene, when you disconnect your kiloview right? But otherwise scene-switching is working fine. It will switch from live to low-bitrate on low-bitrate?

yes, exactly, for example if the kiloview's battery die and It disconnect, the switcher doesn't change from live scene ti brb

**Comment by 715209** [comment]
Hi @FkManu,

You can set the offline trigger to a value greater than 0 to force it to change to the offline scene when the bitrate drops below that number.

**Comment by ibob92** [comment]
I have the same issue, tried to change trigger and to switch to an SRT signal but the problem remains. The script reads that the stream is active at 0kb/s and OBS is stuck to the IRL scene until the signals are manually turned off by the bonding server options.

**Comment by FkManu** [comment]
> Hi @FkManu,
> 
> You can set the offline trigger to a value greater than 0 to force it to change to the offline scene when the bitrate drops below that number.

I tried putting 100 in the offline value it does not change because it does not seem to recognize a signal that arrives but which is exactly 0.
I also tried to do the same thing but deactivating the switch on ''low'' by inserting a value ''null'' but in any case the bot continues to detect the 0 bitrate as a positive value and does not change the scene anyway
